src/json-rpc-handlers/dwn/process-message.ts

```diff
@@ -0,0 +1,103 @@
+import type { Readable } from 'readable-stream';
+import type { JsonRpcHandler, HandlerResponse } from '../../lib/json-rpc-router.js';
+
+import busboy from 'busboy';
+import EventEmitter from 'events';
+
+import { base64url } from 'multiformats/bases/base64';
+import { DataStream } from '@tbd54566975/dwn-sdk-js';
+import { v4 as uuidv4 } from 'uuid';
+
+import { dwn } from '../../dwn.js';
+import { JsonRpcErrorCodes, createJsonRpcErrorResponse, createJsonRpcSuccessResponse } from '../../lib/json-rpc.js';
+
+const emitter = new EventEmitter();
+
+export const handleDwnProcessMessage: JsonRpcHandler = async (dwnRequest, context) => {
+  const requestId = dwnRequest.id ?? uuidv4();
```

Comment:
a request without an `id` is a [notification](https://www.jsonrpc.org/specification#notification).  why are we always assigning a `requestId`?  are we not allowing for notifications in this implementation?

Reply:
yep that's correct. definitely aware of the distinction. will address notification support in a subsequent PR.